src/zarpaint/reader.py

```diff
@@ -9,5 +9,5 @@ def zarr_tensorstore(path: str | pathlib.Path):
     if (str(path).endswith('.zarr') and os.path.isdir(path)
                 and '.zarray' in os.listdir(path)):
         return lambda path: [
-                (open_tensorstore(path), open_ts_meta(path), 'labels')
+                (  zarr.open(path), open_ts_meta(path), 'labels')
```

Comment:
This should be reformatted to follow PEP8. You can use `pip install pre-commit` and then `pre-commit install` in the root directory so that `yapf` is run before each commit and the code is always properly formatted.

Reply:
Thanks, Juan I'll make sure to do sore in future commits :) .

Reply:
Hi Juan 
I've been getting this error every time I try make a commit with pre-commit do you have any idea what the cause could be?
`An unexpected error has occurred: OperationalError: unable to open database file
Failed to write to log at /Users/lencrpe/.cache/pre-commit/pre-commit.log
### version information

```
pre-commit version: 2.20.0
git --version: git version 2.38.1
sys.version:
    3.9.13 | packaged by conda-forge | (main, May 27 2022, 17:00:52) 
    [Clang 13.0.1 ]
sys.executable: /Users/lencrpe/opt/anaconda3/envs/napari-env/bin/python
os.name: posix
sys.platform: darwin
```

### error information

```
An unexpected error has occurred: OperationalError: unable to open database file
```

```
Traceback (most recent call last):
  File "/Users/lencrpe/opt/anaconda3/envs/napari-env/lib/python3.9/site-packages/pre_commit/error_handler.py", line 73, in error_handler
    yield
  File "/Users/lencrpe/opt/anaconda3/envs/napari-env/lib/python3.9/site-packages/pre_commit/main.py", line 386, in main
    return run(args.config, store, args)
  File "/Users/lencrpe/opt/anaconda3/envs/napari-env/lib/python3.9/site-packages/pre_commit/commands/run.py", line 411, in run
    for hook in all_hooks(config, store)
  File "/Users/lencrpe/opt/anaconda3/envs/napari-env/lib/python3.9/site-packages/pre_commit/repository.py", line 227, in all_hooks
    return tuple(
  File "/Users/lencrpe/opt/anaconda3/envs/napari-env/lib/python3.9/site-packages/pre_commit/repository.py", line 230, in <genexpr>
    for hook in _repository_hooks(repo, store, root_config)
  File "/Users/lencrpe/opt/anaconda3/envs/napari-env/lib/python3.9/site-packages/pre_commit/repository.py", line 205, in _repository_hooks
    return _cloned_repository_hooks(repo_config, store, root_config)
  File "/Users/lencrpe/opt/anaconda3/envs/napari-env/lib/python3.9/site-packages/pre_commit/repository.py", line 171, in _cloned_repository_hooks
    manifest_path = os.path.join(store.clone(repo, rev), C.MANIFEST_FILE)
  File "/Users/lencrpe/opt/anaconda3/envs/napari-env/lib/python3.9/site-packages/pre_commit/store.py", line 186, in clone
    return self._new_repo(repo, ref, deps, clone_strategy)
  File "/Users/lencrpe/opt/anaconda3/envs/napari-env/lib/python3.9/site-packages/pre_commit/store.py", line 130, in _new_repo
    result = _get_result()
  File "/Users/lencrpe/opt/anaconda3/envs/napari-env/lib/python3.9/site-packages/pre_commit/store.py", line 123, in _get_result
    with self.connect() as db:
  File "/Users/lencrpe/opt/anaconda3/envs/napari-env/lib/python3.9/contextlib.py", line 119, in __enter__
    return next(self.gen)
  File "/Users/lencrpe/opt/anaconda3/envs/napari-env/lib/python3.9/site-packages/pre_commit/store.py", line 100, in connect
    with contextlib.closing(sqlite3.connect(db_path)) as db:
sqlite3.OperationalError: unable to open database file
```
`